Panic: Unknown type map

I've noticed in the code that the switch statement that deals with types panics when there's a map found, because there's no case for a map.  Why can't I use maps?

```
$ godocjson testone
panic: Unknown type &{Map:572 Key:string Value:string}

goroutine 1 [running]:
main.typeOf(0x546920, 0xc42009c900, 0x0, 0x657490)
	/home/michael/.gopath/src/github.com/rtfd/godocjson/godocjson.go:134 +0xdee
main.processFuncDecl(0xc42009ca80, 0xc4200c6960)
	/home/michael/.gopath/src/github.com/rtfd/godocjson/godocjson.go:141 +0x250
main.CopyFuncs(0xc4200de6e0, 0x4, 0x4, 0xc420010390, 0x7, 0x7ffc01a6f56d, 0x7, 0xc420062240, 0xc42000c220, 0x1, ...)
	/home/michael/.gopath/src/github.com/rtfd/godocjson/godocjson.go:168 +0x85
main.CopyPackage(0xc4200ee000, 0xc420062240, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/michael/.gopath/src/github.com/rtfd/godocjson/godocjson.go:230 +0x6be
main.main()
	/home/michael/.gopath/src/github.com/rtfd/godocjson/godocjson.go:258 +0x284
```

--- a/godocjson.go
+++ b/godocjson.go
@@ -130,6 +130,8 @@ func typeOf(x interface{}) string {
 			results[i] = typeOf(r.Type)
 		}
 		return fmt.Sprintf("func(%s)%s", strings.Join(params, ","), strings.Join(results, ","))
+	case *ast.MapType:
+		return fmt.Sprintf("map [%s]%s", typeOf(x.Key), typeOf(x.Value))
 	default:
 		panic(fmt.Sprintf("Unknown type %+v", x))
 	}

